[ADD] new doc for release report (#1272)
mercadopago/devsite-docs: guides/manage-account/reports/available-money/glossary.en.md

> WARNING
>

> > The Available Balance report will be disabled next March 01, 2022

Reviewer: ```suggestion
> The Available Balance report will be disabled on March 1st, 2022
```

> > The Available Balance report will be disabled from next March 1st, 2022.

Reviewer: ```suggestion
> The Available Balance report will be disabled from March 1st, 2022.
```

>

> > Use the [Releases report](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/manage-account/reports/released-money/introduction) to reconcile the transactions that affect the balance available in your account, including your bank withdrawals.

Reviewer: ```suggestion
> You can use the [Releases report](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/manage-account/reports/released-money/introduction) to reconcile the transactions that affect the balance available in your account, including your bank withdrawals.
```
